executor/compiler.go

```diff
@@ -32,6 +34,14 @@ type Compiler struct {
 // a plan, and we wrap the plan in an adapter as stmt.Statement.
 // If it is not supported, the node will be converted to old statement.
 func (c *Compiler) Compile(ctx context.Context, node ast.StmtNode) (ast.Statement, error) {
+	var stmtName string
+	if stringer, ok := node.(fmt.Stringer); ok {
```

Comment:
okay...I'll remove `String` method.
BTW, print the detail I think we can use `GoString`, which is called by fmt.Printf "%#v", `String` should be brief.